release/likely-commonjs.js

```diff
@@ -1422,11 +1426,8 @@ var Likely = function () {
     }, {
         key: 'ready',
         value: function ready() {
-            if (this.timeout) {
-                clearTimeout(this.timeout);
-
-                this.container.classList.add(__WEBPACK_IMPORTED_MODULE_1__config__["default"].name + '_ready');
-            }
+            clearTimeout(this.timeout);
+            this.container.classList.add(__WEBPACK_IMPORTED_MODULE_1__config__["default"].name + '_ready');
```

Comment:
If counters are disabled there's no timeout set but we still need to be able to set `_ready` modifier (via calling `ready` function).
We don't need `if (this.timeout)` check at all, since if there's no timeout set it's fine to call clearTimeout. No exceptions.

Reply:
makes sense